examples/chip-tool/commands/common/ModelCommand.cpp

```diff
@@ -109,20 +107,30 @@ void ModelCommand::ReceiveCommandResponse(ChipDeviceController * dc, PacketBuffe
     commandId      = chip::Encoding::Read8(message);
     messageLen -= 3;
 
-    VerifyOrExit(frameControl == 8, ChipLogError(chipTool, "Unexpected frame control byte: 0x%02x", frameControl));
+    // frameControl 8 is for global cmds and frameControl 9 is for specific cluster
+    VerifyOrExit((frameControl == 8 || frameControl == 9),
+                 ChipLogError(chipTool, "Unexpected frame control byte: 0x%02x", frameControl));
     VerifyOrExit(sequenceNumber == 1, ChipLogError(chipTool, "Unexpected sequence number: %d", sequenceNumber));
 
-    switch (commandId)
+    isGlobalCommand = (frameControl == 8);
+    if (isGlobalCommand)
+    {
+        switch (commandId)
+        {
+        case 0x0b:
```

Comment:
while we are touching this bit, could we add constants for the 0x0b and 0x01? They look so magical.

same for frameControl 8 and 9 above: we apparently needed comments to say what 8 and 9 mean.

Reply:
Will do.

Reply:
> approving, pending @andy31415 's suggestions

Made suggested changes.